frontend/src/features/documents/list/DocumentListView.test.tsx

```diff
@@ -34,6 +36,7 @@ describe('Document List View', () => {
         parentId={0}
         relationshipType={DocumentRelationshipType.FILES}
         documentResults={mockDocumentsResponse()}
+        deleteHandle={deleteMock}
```

Comment:
please use `renderProps.deleteMock` instead to support overriding it with passed props (as shown in tests below)

Reply:
Also, don't see any relevant unit test to cover the delete document scenario

Reply:
There are almost none at this point. The reason is that is changing so quickly that I don't think it would be good to add tests until it gets into a more stable point.